charts/tidb-cluster/templates/tidb-secret.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.clusterName }}-tidb
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Values.clusterName }}
+    app.kubernetes.io/component: tidb
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+type: Opaque
+data:
+  {{- if .Values.tidb.password }}
+  password: {{ printf "SET PASSWORD FOR 'root'@'%%' = '%s' ; FLUSH PRIVILEGES;" .Values.tidb.password | b64enc }}
```

Comment:
I would much prefer to have *just* the password as the secret.

Reply:
You can see the reasons why I store the sql command here instead of only password.
https://github.com/pingcap/tidb-operator/pull/171/files#diff-7b1a419604c0fcc0219f5b547c56eb40R27

Reply:
The non-secret part could be a ConfigMap then. Encrypting the SQL means that editing this SQL requires accessing the password.

Reply:
For other SQL initialization setups, I think we can store them in another ConfigMap instead of storing them in this Secret.